### ISSUE TITLE
Search words by morpheme

### ISSUE THREAD

**Issue by eddieantonio** [UAlbertaALTLab/cree-intelligent-dictionary#726]
I want to search for words by their derivational morphemes.

This would involve:

 - [ ] importing morphemes in to the database from their sources (in this repo? https://github.com/UAlbertaALTLab/dictionary-database)
 - [ ] indexing each head by its morpheme
 - [ ] adding a search function that checks if the queries is a morpheme, and augments the search results with all of the heads that contain this morpheme

For example, I want to search words with `kisk-` in them (memory, recollection, learning, etc.), and it should return
 - kiskisiw
 - kiskêyihtâsowin
 - kiskêýihtam
 - kiskahikêw
 - etc.

(every word with /-êýiht/ would be awesome too!)

I would also LOVE to have definitions for morphemes (derivational morphemes and roots especially!), but I think that's a different issue.

Currently, the Carleton infrastructure does this, however the results are a bit hard to parse.

**Comment by dwhieb** [comment]
Sadly the CW database does not include definitions for morphemes. However, by pulling them out into separate entries in the ALTLab dictionary database, we're enabling Arok to provide this data more quickly/easily in the future.

**Comment by aarppe** [comment]
This has been also discussed in #391.

**Comment by aarppe** [comment]
In the interim, we could incorporate the morpheme definitions for several hundred cases as available in the Cook & Mühlbauer list.

**Comment by aarppe** [comment]
A search string could be recognized as implicitly concerning a morpheme by the presence of a hyphen 1) before the string (`-morf` -> _final_), 2) after the string (`morf-` -> _initial_), or both before and after the string (`-morf-` -> _medial_). Alternatively, the flag `morf` could be use to toggle this behavior, i.e. as `morf:1` or `morf:t`.
